pavelib/quality.py

```diff
@@ -147,10 +138,42 @@ def run_pylint(options):
     with open(Env.METRICS_DIR / "pylint", "w") as f:
         f.write(violations_count_str)
 
-    # Fail number of violations is greater than the limit
-    if num_violations > violations_limit > -1:
-        raise BuildFailure("Failed. Too many pylint violations. "
-                           "The limit is {violations_limit}.".format(violations_limit=violations_limit))
+    # Fail when number of violations is less than the lower limit,
+    # which likely means that pylint did not run successfully.
+    # If pylint *did* run successfully, then great! Modify the lower limit.
+    if num_violations < lower_violations_limit > -1:
+        raise BuildFailure(
+            "Failed. Too few pylint violations. "
+            "Expected to see at least {lower_limit} pylint violations. "
+            "Did pylint run successfully?".format(lower_limit=lower_violations_limit)
```

Comment:
The failure message is more likely to be seen than the comment above, so maybe change "Did pylint run successfully?" to "Pylint may not be running correctly, or it may be time to lower thresholds."

Reply:
Done - reworded the message.